go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/stretchr/testify v1.7.2
-	github.com/waku-org/go-zerokit-rln-apple v0.0.0-20230916172309-ee0ee61dde2b
+	github.com/waku-org/go-zerokit-rln-apple v0.0.0-20240116121347-ee5a1d931442
```

Comment:
Temporal, points to https://github.com/waku-org/go-zerokit-rln-apple/pull/4
For testing.